clients/docusaurus-theme-search/README.md

```diff
@@ -0,0 +1,43 @@
+## Adding into your site
+
+Add the following into your `docusaurus.config.js`
+
+
+```js docusaurus.config.js
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  ...
+  themes: [
+    [
+      require.resolve("@getcanary/docusaurus-theme-search-pagefind"),
+      {
+        apiKey: "tr-****************",
+        datasetId: "<your-dataset-id>"
+        defaultSearchQueries: [
+          "How to use the API reference?",
+          "Is there a python sdk?",
+          "How to get started?"
+        ]
+      }
+    ],
+  ],
+};
+```
```

Comment:
Example configuration, any other json fields can also be passed into this (unsure about the JSX features though)